### ISSUE TITLE
[FR] Hide Terminal and only show guiscrcpy

### ISSUE THREAD

**Issue by PhoenixCODE2322** [srevinsaju/guiscrcpy#324]
is it possible to hide the terminal , it seems unnecessary and clutters the user experience

**Additional context**
![image](https://user-images.githubusercontent.com/75798544/185636805-e6a687ee-5ee4-4495-ba14-12541885dfbb.png)

**Comment by sperglord8008s** [comment]
I Agree with this suggestion heavily.

**Comment by srevinsaju** [comment]
Yes, its a technical constraint. This is a limitation on Windows alone, the packaging system for converting Python code to Windows executables requires a open console for some reason to locate resources. There is a `noconsole` option for PyInstaller, but it did not work the last time I tested it (a year ago), but I am interested if anyone would like to test it. https://stackoverflow.com/questions/17584698/getting-rid-of-console-output-when-freezing-python-programs-using-pyinstaller
